app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,12 @@
   </head>
 
   <body>
+  <% if notice %>
+  <p class="alert alert-success"><%= notice %></p>
```

Comment:
@amitshethe Indentations seems incorrect

Reply:
corrected them in controller setup branch